vnext/ReactCommon/TEMP_UntilReactCommonUpdate/react/renderer/mapbuffer/MapBuffer.h

```diff
@@ -89,8 +89,9 @@ class MapBuffer {
     Bucket(Key key, uint16_t type, uint64_t data)
         : key(key), type(type), data(data) {}
   };
+  #pragma pack(pop)
 
-  static_assert(sizeof(Header) == 8, "MapBuffer header size is incorrect.");
+  static_assert(sizeof(Header) == 12, "MapBuffer header size is incorrect.");
```

Comment:
size_t is a different size on different architectures. It's effectively a uint32_t on 32-bit machines, and a uint64_t on 64 bit machines.